src/legacy/core_plugins/apm_oss/index.js

```diff
@@ -38,7 +38,7 @@ export default function apmOss(kibana) {
         spanIndices: Joi.string().default('apm-*'),
         metricsIndices: Joi.string().default('apm-*'),
         onboardingIndices: Joi.string().default('apm-*'),
-        apmAgentConfigurationIndex: Joi.string().default('.apm-agent-configuration')
+        apmAgentConfigurationIndex: Joi.string().default('.apm-agent-configuration'),
```

Comment:
I wonder if this should have been in the OSS version. Probably not.

Reply:
Btw. now you are at it, can you add this to the docs like you did with `maxTraceItems`?